src/go/k8s/tests/e2e/resource-redpanda-create/00-assert.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: redpanda.vectorized.io/v1alpha1
+kind: Redpanda
+metadata:
+  finalizers:
+    - operator.redpanda.com/finalizer
+  name: redpanda
+spec:
+  chartRef:
+    chartVersion: ""
+  clusterSpec:
+    statefulset:
+      replicas: 1
+status:
+  conditions:
+    - message: Redpanda reconciliation succeeded
+      reason: RedpandaClusterDeployed
+      status: "True"
+      type: Ready
+  helmRelease: redpanda
+  helmRepository: redpanda-repository
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: redpanda
+spec:
+  replicas: 1
+status:
+  availableReplicas: 1
+  currentReplicas: 1
+  readyReplicas: 1
+  replicas: 1
+  updatedReplicas: 1
+---
+
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+collectors:
+  - type: pod
+    selector: app.kubernetes.io/name=redpanda
+    tail: -1
```

Comment:
I personally would not use collectors as this pollutes stdout of the kuttle cli, but it's very hard to find the root cause. I would advise to do something like:

https://github.com/redpanda-data/redpanda/blob/8a9d43faa4b0d998eb6cd79521090b1733eca0e1/src/go/k8s/tests/e2e/update-image-tls/00-assert.yaml#L59-L70

I know it's painful to maintain the paths and names, but I didn't check the env vars during collectors. Maybe there is options to make collectors more generic.

Reply:
I see, thank you for that!